src/app/screens/Login/index.test.tsx

```diff
@@ -37,4 +44,18 @@ describe('Login test', () => {
     userEvent.click(screen.getByText('Login:login'));
     await waitFor(() => expect(screen.getAllByRole('alert').length).toBe(MAX_ALERTS_LENGTH));
   });
+
+  test('Local storage should be called after sucess login request.', async () => {
+    server.use(
+      rest.post(`${API_URL}`, (req, res, ctx) =>
+        res(ctx.status(STATUS_CREATED), ctx.json(OK_LOGIN_API_RESPONSE))
+      )
+    );
```

Comment:
the `setupServer` receives an array of handlers, `rest.post(...` is a handler. 
The purpose of this is to not call server anywhere, just in `setupTests`